skops/card/_model_card.py

```diff
@@ -373,6 +374,34 @@ def add_metrics(self, **kwargs: str) -> "Card":
             self._eval_results[metric] = value
         return self
 
+    def add_feature_importances(self, feature_importances, data) -> "Card":
+        """Visualize permutation importance.
+
+        Parameters
+        ----------
+        feature_importances : sklearn.utils.Bunch
+            Output of sklearn.inspection.permutation_importance()
+
+        data :
```

Comment:
Here it looks like `data` has to be a pandas df. But in the end, only the column names are used, right? So how about we just pass the column names here? Also, I would make the argument optional, in case people only have a numpy array. Then the features would just be called `x0`, `x1`, etc. WDYT?

Reply:
I changed this to have columns instead.